src/Symfony/Component/BrowserKit/CookieJar.php

```diff
@@ -33,13 +33,9 @@ public function set(Cookie $cookie)
      * (this behavior ensures a BC behavior with previous versions of
      * Symfony).
      *
-     * @param string $name   The cookie name
-     * @param string $path   The cookie path
-     * @param string $domain The cookie domain
-     *
      * @return Cookie|null A Cookie instance or null if the cookie does not exist
      */
-    public function get($name, $path = '/', $domain = null)
+    public function get(string $name, string $path = '/', string $domain = null)
```

Comment:
Since the `$domain` parameter accepts null, the correct typehint here should be ?string

```suggestion
    public function get(string $name, string $path = '/', ?string $domain = null)
```